Problems running 2 Pico Flexx at the same time.

Tested on Ubuntu 16.04 + ROS Kinetic

I'm running 2 different Pico Flexx sensors on the same machine. Individually, they work correctly, but I get a 
`[PicoFlexx::selectCamera] no cameras connected!` from one of the nodelets when running 2 copies of the driver with correctly set sensor ids. The other one works correctly. I'm running 2 copies of the driver in 2 separate nodelet managers.

@mintar: Thanks for the quick replies! It took me a bit to get back to testing the sensors.

Running both nodelets in the same nodelet manager worked! I've verified streaming of both pointclouds in rviz. I haven't looked at the libroyale code, but it looks like only one copy per process can grab the sensors.

There does seem to be a dynamic reconfigure setup clash, since the dynamic reconfigure servers are not currently namespaced by the nodelet itself. I still need to look into this in more detail. Some trivial attempts at remapping failed:
```
[ERROR] [1510846252.056941608]: Tried to advertise a service that is already advertised in this node [/robutler/pico_flexx_manager/set_parameters]
```
I'll leave this open for now. I'll try and figure out a fix for the reconfigure clash without without affecting backwards compatibility. If you have any suggestions for the problem, let me know!

--- a/src/pico_flexx_driver.cpp
+++ b/src/pico_flexx_driver.cpp
@@ -137,7 +137,7 @@ private:
 
 public:
   PicoFlexx(const ros::NodeHandle &nh = ros::NodeHandle(), const ros::NodeHandle &priv_nh = ros::NodeHandle("~"))
-    : royale::IDepthDataListener(), royale::IExposureListener2(), nh(nh), priv_nh(priv_nh), server(lockServer)
+    : royale::IDepthDataListener(), royale::IExposureListener2(), nh(nh), priv_nh(priv_nh), server(lockServer, priv_nh)
   {
     cbExposureTime.resize(2);
     running = false;

